toctoc/slickMySql/src/main/scala/io.buildo.toctoc/slick/authentication/token/MySqlSlickAccessTokenAuthenticationDomain.scala

```diff
@@ -24,14 +25,13 @@ class MySqlSlickAccessTokenAuthenticationDomain[F[_]: FutureLift[?[_], Future]](
   implicit F: Sync[F],
 ) extends AccessTokenDomain[F] {
 
-  import MySqlSlickHelper._
-
   class AccessTokenTable(tag: Tag)
-      extends Table[(Int, String, String, Instant)](tag, schemaName, tableName) {
+      extends Table[(Int, String, String, Timestamp)](tag, schemaName, tableName) {
     def id = column[Int]("id", O.PrimaryKey, O.AutoInc)
     def ref = column[String]("ref")
     def token = column[String]("token", O.Length(255))
-    def expiresAt = column[Instant]("expires_at")
+    def expiresAt =
+      column[Timestamp]("expires_at", O.SqlType("timestamp not null default CURRENT_TIMESTAMP"))
```

Comment:
The `SqlType` excludes the column option `on update CURRENT_TIMESTAMP` during table generation. That option is not good for a token expiration column.

Reply:
As discussed offline, let's keep this concern for another issue.